shader/coarse.wgsl

```diff
@@ -401,6 +410,9 @@ fn main(
     }
     if bin_tile_x + tile_x < config.width_in_tiles && bin_tile_y + tile_y < config.height_in_tiles {
         ptcl[cmd_offset] = CMD_END;
-        // TODO: blend stack allocation
+        if max_blend_depth > BLEND_STACK_SPLIT {
+            let scratch_size = max_blend_depth * TILE_WIDTH * TILE_HEIGHT * 4u;
```

Comment:
What's the `4u` for? Atomic bump allocation is in elements (u32 here) in Vello but was bytes in piet-gpu.

I guess there's no other code that depends on this yet, so it could go either way, but generally prefer consistency.

Reply:
Good catch, ty.